packages/fcl/src/current-user/index.js

```diff
@@ -301,5 +308,6 @@ currentUser.signUserMessage = signUserMessage
 currentUser.verifyUserSignatures = verifyUserSignatures
 currentUser.subscribe = subscribe
 currentUser.snapshot = snapshot
+currentUser.resolve = resolveArgument
```

Comment:
This needs to be something like `currentUser.resolveArgument`, lots of things can be resolved and we need this to be more specific. Once it is picked up and in the scope of being processed as an argument it could then be called resolve internally, but externally it should be something like `resolveArguments`.

Take the following example

```javascript
mutate({
  cadence: `
    transaction(addr: Address) {
      prepare(currentUser: AuthAccount) { ... }
    }
  `,
  args: () => [currentUser], // uses resolveArgument
  authz: currentUser,           // uses resolveAuthz
})
```

The way you have done this will make passing currentUser directly into a transaction as the authorization function more difficulat because that already has a concept of `authz.resolve` similar to the arugments having an `arg.resolve`